### PR TITLE
Support Realm 10 and bumped version to 4.0.0

### PR DESCRIPTION
**Pull request by iDevid**

### CODE REVIEW

**Comment by tristangrichard** [comment]
Any chance of reviewing this ASAP?

File: RxRealm.podspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name             = "RxRealm"
-  s.version          = "3.1.0"
+  s.version          = "3.2.0"
```
Comment (freak4pc) [inline RxRealm.podspec]: might want this to be a major 4.0,0 bump

Comment (iDevid) [reply RxRealm.podspec]: done

**Comment by iDevid** [comment]
@freak4pc some news about this?

**Pull request merged**